samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Bookmarks.razor

```diff
@@ -0,0 +1,53 @@
+@page "/Bookmarks"
+@using dymaptic.GeoBlazor.Core.Components.Widgets;
+
+<PageTitle>Bookmarks and Bookmarks Widget</PageTitle>
+<h1>Bookmarks Widget</h1>
+
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Bookmarks.html">ArcGIS API for JavaScript Reference</a>
+    <a class="btn btn-primary" target="_blank" href="https://www.arcgis.com/home/item.html?id=70b726074af04a7e9839d8a07f64c039">Bookmarks widget</a>
+</div>
+
+<p class="instructions">
+    Sample demonstrates the use of the Bookmarks widget and Bookmark actions.
+</p>
+
+<div class="form-group">
+    <label>Show Bookmarks and Bookmarks Widget: <input type="checkbox" class="form-check-input" @onchange="@(()=> _showBookmarks = !_showBookmarks)"></label>
+</div>
+
+<MapView @ref="MapView" Class="map-view">
+    <Map>
+        <Basemap>           
+            <PortalItem Id="70b726074af04a7e9839d8a07f64c039" />
+        </Basemap>
+    </Map>
+    @if (_showBookmarks)
+    {
+        <ExpandWidget ContainerId="BookmarkDiv" Position="OverlayPosition.TopRight">
+            <div class="container">
+                <div class="row">
+                    <div class="col-12">
+                        <p>Click on a bookmark to navigate to its location.</p>
+                        <BookmarksWidget  />
+                    </div>
```

Comment:
Is this page not finished? Or does the BookmarksWidget really work with no parameters?